tests/test_routes.py

```diff
@@ -322,9 +327,10 @@ def test_create_with_invalid_content_type(self):
 
 class TestYourResourceServerUpdate(TestResourceServer):
     """Test Cases for Inventory Resource Server Update"""
+
     def test_update_normally(self):
         """Update normally"""
-        test_inventory = InventoryFactory(condition=Condition.NEW)
+        test_inventory = InventoryFactory(product_id=1, condition=Condition.NEW)
```

Comment:
IIRC, I was getting serialization issues with regards to the product_id being "None"

Specifying a product_id fixed it